app/main.py

```diff
@@ -2,5 +2,16 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    stored_kwargs = {}
+
+    def stored(*args, **kwargs) -> Callable:
+        checked_kwargs = tuple(kwargs.items())
+        checked = (args, checked_kwargs)
```

Comment:
It is a good approach to save function name here as well to handle case when functions have same args